src/components/helpers/handleReaquestNotification.js

```diff
@@ -0,0 +1,15 @@
+import 'react-toastify/dist/ReactToastify.css'
+import { toast } from 'react-toastify'
+
+export const handleRequestError = ({ message }) => {
```

Comment:
please remove returns at first
at second, error it's danger object, can be situation when it will be null
so you will have error here.

Most better:
```js
export const handleRequestError = (error) => 
       toast.error(
              error?.mesagge || "Developers forgot to handle this error",
              { position: toast.POSITION.TOP_LEFT }
       )
```

Reply:
done